config/config.go

```diff
@@ -28,6 +29,11 @@ type Config struct {
 		MinimumPinEntropy float64 `yaml:"minimumPinEntropy,omitempty"`
 		// AllowMonotonousPins will allow Pins like 123456, 122334 or 654321, 662211, ...
 		AllowMonotonicPins bool `yaml:"allowMonotonicPins,omitempty"`
+		AllowedFromTime *int `yaml:"allowedFromTime,omitempty"`
+		AllowedUntilTime *int `yaml:"allowedUntilTime,omitempty"`
+		AllowedFromDate *strfmt.DateTime `yaml:"allowedFromDate,omitempty"`
+		AllowedUntilDate *strfmt.DateTime `yaml:"allowedUntilDate,omitempty"`
```

Comment:
What is the use case of a global configuration for the allowed*Dates?

Reply:
yes, it's more a per person option. 
can be removed